### PR TITLE
Update `README.md`.

### PR DESCRIPTION
**Pull request by JSAssassin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,87 @@
...
+secrets, please check the implementation manifest within the
+[vc-api-test-suite-implementations](https://github.com/w3c-ccg/vc-api-test-suite-implementations/tree/main/implementations) library.
+
+### Docker Integration (TODO)
+
+We are presently working on implementing a new feature that will enable the
+utilization of Docker images (using the VC API mentioned above) instead of
+live endpoints.
+
+## License
```
Comment (aljones15) [inline README.md:79]: I'm not sure we need the license stuff here @msporny is that correct?

File: README.md [hunk 2]
```diff
@@ -1,5 +1,87 @@
...
+
+## Usage
+
+```
+npm test
+```
+
+## Implementation
+
+You will need an issuer and verifier that are compatible with [VC API](https://w3c-ccg.github.io/vc-api/)
```
Comment (aljones15) [inline README.md:32]: this looks amazing and thanks

**Review by aljones15 — APPROVED** [review]
I'm approving, but get confirmation from @msporny on if we want the copyright stuff at the end of the README to be there.

File: README.md (hunk 1 above)
Comment (msporny) [reply README.md:81]: Yes, we need the license on here because we're transferring this to W3C.

**Review by msporny — APPROVED** [review]

**Comment by TallTed** [comment]
Merged too quickly! Creating another PR is a much heavier lift than a change suggestion on this PR. Please always allow at least two business days for others to review!

**Pull request merged**